Source/WebCore/css/ComputedStyleExtractor.cpp

```diff
@@ -671,6 +671,11 @@ static Ref<CSSValueList> borderRadiusShorthandValue(const RenderStyle& style)
 
     if (!verticalRadiiList->equals(downcast<CSSValueList>(*list->item(0))))
         list->append(WTFMove(verticalRadiiList));
+    else if (propertyID == CSSPropertyWebkitBorderRadius) {
+        auto* horizontals = downcast<CSSValueList>(list->item(0));
+        if (horizontals->length() == 2)
```

Comment:
Can we base this check on the Boolean locals instead of reading back the length of the list?